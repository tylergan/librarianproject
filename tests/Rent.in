rent 100000 111111
add member John
rent 100000 111111
add book books.csv 111111
list all long
list available
rent 100001 222222
rent 100000 222222
rent 100000 111111
list all long
list available
exit