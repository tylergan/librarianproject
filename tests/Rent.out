user: No members in system.

user: Success.

user: No books in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

user: To Kill a Mockingbird (Harper Lee)

user: No such member in system.

user: No such book in system.

user: Success.

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Rented by: 100000.

user: No books available.

user: Ending Library process.
